backend/models/base.py

```diff
@@ -1,47 +1,77 @@
 """Base model and other useful tools for project models."""
-from typing import Annotated, Final, TypeAlias, TypedDict, TypeVar, cast
+from contextlib import suppress
+from dataclasses import dataclass
+from typing import Annotated, ClassVar, Final, TypeAlias, TypedDict, TypeVar, cast
 
-from desert import schema
+# from desert import schema
 from marshmallow import Schema as MMSchema
 from models.mixins.repr import ReprMixin
 from models.mixins.serialize import SerializeMixin
 from sqlalchemy import Column, DateTime, ForeignKey, func
-from sqlalchemy.orm import declarative_base, declared_attr, mapped_column, registry
+from sqlalchemy.orm import Mapped, declarative_base, declared_attr, mapped_column, registry
 from typing_extensions import Any, override
 
 Base = declarative_base()
 reg = registry(metadata=Base.metadata)
 
-intpk = Annotated[int, mapped_column(init=False, primary_key=True)]
+intpk = Annotated[int, mapped_column(init=False, repr=True, primary_key=True)]
 # This is a simple type to make a standard int-base primary key field.
 
 _T = TypeVar("_T")
-_schema_registry: dict[str, MMSchema] = {}
+# _schema_registry: dict[str, MMSchema] = {}
```

Comment:
Please remove commented code

Reply:
Hmm..... ok. I kept it in case it was possible to get desert to work, but I can remove the commented code.

Reply:
It'll exist in the commit history still - or save to a WIP branch if needed.

Reply:
Fixed